### ISSUE TITLE
can´t download xpi file

### ISSUE THREAD

**Issue by henryECU** [willsALMANJ/Zutilo#233]
Hi 
Please verify release page because this error appear
![Screenshot_20220720_091715](https://user-images.githubusercontent.com/97720524/180007290-0167e9bb-b4a4-4196-a509-0128aed1ebc2.png)
:

**Comment by qqobb** [comment]
Please make sure to read through https://github.com/wshanks/Zutilo#installation.

**Comment by henryECU** [comment]
Yes. I read that document. It says there to download the .xpi file. It is precisely what cannot be done because of the error that I have sent before. Which is a gitHub post.
Please correct it.

**Comment by wshanks** [comment]
The "Note for Firefox Users" section below where @qqobb linked to explains what is going in your screenshot. Firefox is trying to install the file as a Firefox add-on (in the past, Zotero was a Firefox add-on and plugins to Zotero were separate Firefox add-ons, but now Zotero only supports standalone but still uses xpi for its plugins). I have seen the same issue in the past where I could find no way for Firefox not to give me that dialog. That is why the note suggests using `curl`.

However, at the moment, right-click->"Save as" is working for me using Firefox 102 on Linux. Are you doing that that or are you left-clicking on the xpi link? If I left-click, I get the same dialog as you showed above.

**Comment by henryECU** [comment]
But the message is from gitHub saying it's a corrupt file.
I use Konqueror and there it does not download and no message appears.
Perhaps it could be downloaded with another extension, not .xpi, and create a script to change the name once downloaded.

**Comment by wshanks** [comment]
No, the message is regarding `github.com` but it is generated by Firefox, not by the GitHub web page itself. The message indicates that Firefox is trying to install Zutilo as a Firefox add-on but Zotero uses an older XPI format that Firefox no longer supports so Firefox says that it is corrupt.

I tried in  Konqueror just now. I get a dialog window as shown in this screenshot from which I can select to save the file. Right-clicking and choose "Save Link As..." also works for me in Konqueror.

![image](https://user-images.githubusercontent.com/1178780/180216432-40a9ceca-8989-4522-8445-e9cc081adc8a.png)

**Comment by henryECU** [comment]
Yes, konqueror is great.
If there were no such problem, more people would download your app.
